src/goto-analyzer/static_instrument.h

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************\
+
+Module: goto-analyzer
+
+Author: Martin Brain, martin.brain@cs.ox.ac.uk
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_ANALYZER_STATIC_INSTRUMENT_H
+#define CPROVER_GOTO_ANALYZER_STATIC_INSTRUMENT_H
+
+#include <iosfwd>
+
+// What kind of instrumentation to add
+enum class instrument_actiont
```

Comment:
We are in contact and the goal is to fit with and support @remi-delmas-3000 's work.  Thanks for the link though, if I didn't know about what he was doing it would be very important to find out!